package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "novo-elements",
-  "version": "6.0.0",
+  "version": "6.0.1",
```

Comment:
This should be done by the release action

Reply:
reverted, thanks